src/main/java/com/autotune/common/data/metrics/Metric.java

```diff
@@ -76,6 +78,7 @@ public String getValueType() {
         return valueType;
     }
 
+    @JsonProperty(KruizeConstants.JSONKeys.RESULTS)
```

Comment:
This should be `metrics` and not `results`

Reply:
@dinogun Please correct me if I'm wrong, I see the var specific to that getter is mentioned to match `results`

```
@SerializedName("results")
    private MetricResults metricResults;
```

So I have added the JSON property as `results`

Reply:
Sorry yes, ignore above comment